contracts/credit/CreditManagerV3.sol

```diff
@@ -913,8 +901,8 @@ contract CreditManagerV3 is ICreditManagerV3, SanityCheckTrait, ReentrancyGuardT
         returns (uint256 tokensToEnable, uint256 tokensToDisable)
     {
         CreditAccountInfo storage currentCreditAccountInfo = creditAccountInfo[creditAccount];
-        if (quotaChange > 0 && currentCreditAccountInfo.debt == 0) {
-            revert IncreaseQuotaOnZeroDebtAccountException();
+        if (currentCreditAccountInfo.debt == 0) {
+            revert UpdateQuotaOnZeroDebtAccountException();
```

Comment:
In theory that doesn't change anything since zero-debt account must have no quotas anyways, but it looks more robust this way.